src/html-conventions-lib/dependency-html-conventions.xsl

```diff
@@ -105,6 +105,13 @@
                     <xsl:call-template name="dependencySourceTargetTypes">
                         <xsl:with-param name="dependencyConnector" select="."/>
                     </xsl:call-template>
+                    <xsl:call-template name="connectorUniqueName">
+                        <xsl:with-param name="connector" select="."/>
+                    </xsl:call-template> 
+                    <xsl:call-template name="connectorRoleCheck">
+                        <xsl:with-param name="connector" select="."/>
+                        <xsl:with-param name="isDependency" select="fn:true()"/>
```

Comment:
Looks like you have two logics in one function. 
Please get rid of the "isDependecy" flag and break down the logic into two sub-functionalities.

Reply:
if the documentation will reflect that this is not a common checker will happily break it into 2 specific checkers. For now I think this should stay in the common if the documentation is showing that this checker is inherited. Let me know if this will appear as specific checker